docs/getting-started/index.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Quickstart
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Quickstart
+
+---
+
+This quickstart will use the [ngrok agent](/agent/) to put your application on
+the internet and secure it so that only you can access it.
```

Comment:
the internet and secure it so that only you can access it. It assumes you already have your
own application running locally.